iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -11,45 +11,45 @@ import shared
 
 struct AddMealView: View {
 
-    @ObservedObject private(set) var viewModel: ViewModel
+    @ObservedObject private(set) var viewModel: AddMealViewModel
 
     var body: some View {
-        //TODO: Pass the actual values when UI is complete
-        Button("Add a meal") {
-            let meal = Meal(
-                id: "\(viewModel.sdk.getUUID())",
-                name: "Pizza",
-                quantity: 2,
-                availableFromDate: "Tuesday",
-                expiryDate: "Saturday",
-                info: "Meat",
-                hot: false,
-                locationLat: 51.509865,
-                locationLong:  -0.118092)
-            self.viewModel.postMeal(meal: meal)
+        ZStack {}
+        .alert(isPresented: $viewModel.showingCollectionCode) {
```

Comment:
SwiftUI gets upset if you have two alerts on the same view so this is just here temporarily.
When we get to the stage of showing something more elaborate, we can then get rid of this ZStack and alert code.